Sources_v3/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -294,6 +294,25 @@ class MessageUpdater<ExtraData: ExtraDataTypes>: Worker {
             attachmentDTO.localState = .pendingUpload
         }, completion: completion)
     }
+
+    /// Updates local state of attachment with provided `id` to be enqueued by attachment uploader.
+    /// - Parameters:
+    ///   - id: The attachment identifier.
+    ///   - completion: Called when the attachment database entity is updated. Called with `Error` if update fails.
```

Comment:
Copy/Paste error in the comment: attachment -> message

Reply:
fixed in ed6cf54d6967cb31422b2ad952a39e052cb855e4